build.sbt

```diff
@@ -94,22 +94,25 @@ lazy val core = (crossProject(JSPlatform, JVMPlatform) in file ("core"))
       logback             %   "test",
       "org.scalacheck"    %%% "scalacheck"      % scalacheckVersion              % "test",
       "org.scalatest"     %%% "scalatest"       % scalatestVersion               % "test",
-      "org.scalatestplus" %%% "scalacheck-1-14" % scalatestPlusScalacheckVersion % "test",
-      reflect.value       %   "provided"
+      "org.scalatestplus" %%% "scalacheck-1-15" % scalatestPlusScalacheckVersion % "test",
```

Comment:
* Blocker: not published on 3.0.0-M1.
* Blocker: not published for scala.js on 0.27.

Reply:
It's since published for 3.0.0-M1, and we dropped 0.27.